<head>
    <title>Wikinotes</title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.0.0/semantic.min.css" />
    <link rel="stylesheet" href="/static/styles.css" />
    <meta name="viewport" content="width=device-width">
    
<script type="text/javascript"
        src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
    TeX: {
        extensions: ['cancel.js']
    },
    tex2jax: {
        inlineMath: [  ['$', '$'] ],
        processEscapes: true
    }
});
</script>

</head>
<body>
    
    <div id="header" class="ui container">
        <a href="/">
            <img src="/static/img/logo-header.png" class="ui image" />
        </a>
    </div>
    
    <div id="content">
        <div class="ui container">
            
<div class="ui container">
    <div class="ui secondary segment">
        <div class="ui large breadcrumb">
            <a class="section" href="/">Home</a>
            <i class="right chevron icon divider"></i>
            <a class="section" href="/BIOL_112/">
                BIOL 112
            </a>
            <i class="right chevron icon divider"></i>
            <span class="active section">
                
                Notes for Part I: The Cell (Vogel&#x27;s Section)
                
            </span>
        </div>
    </div>
    <h1 class="ui header">
        <div class="content">
            
            Notes for Part I: The Cell (Vogel&#x27;s Section)
            
            <span>
                <a href="http://creativecommons.org/licenses/by-nc/3.0/">
                    <img src="/static/img/cc-by-nc.png" alt="CC-BY-NC"
                         title="Available under a Creative Commons Attribution-NonCommercial 3.0 Unported License" />
                </a>
            </span>
            
        </div>
    </h1>
    <div class="ui icon list">
        <div class="item">
            <i class="user icon"></i>
            <div class="content">
                <strong>Maintainer:</strong> admin
            </div>
        </div>
    </div>
    <div class="ui divider"></div>
    <div id="wiki-content">
	
        <p><a href="/BIOL_112/winter-2011">BIOL 112 (Winter 2011)</a> notes for <strong>Part I: The cell</strong> (Professor Vogel's section). From January 5, 2011 to February 11, 2011.</p>
<h2 class="header"><i>1</i>Small molecules<a class="headerlink" href="#small-molecules" name="small-molecules">&para;</a></h2>
<ul>
<li>The below is mostly review of basic chemistry</li>
<li>Atomic structure: protons, neutrons, electrons</li>
<li>Mass of proton = standard unit of measure called the dalton OR amu (1 amu is roughly 1.7 x 10<sup>-24</sup> g)</li>
<li>Matter: anything that occupies space, and has mass (which is a quantity of matter, circular definitions ftw)</li>
<li>Inertia: resistance to change of state of motion</li>
<li>Classification of matter:<ul>
<li>Separable by physical means = mixture<ul>
<li>Homogeneous (uniform throughout, e.g. salt solution) or heterogeneous (not, e.g. milk - fat in a water-based medium)</li>
</ul>
</li>
<li>Not separable by physical means = substance<ul>
<li>Compound (decomposable by chemical process) or element (not)</li>
</ul>
</li>
</ul>
</li>
<li>Atomic mass = mass of neutrons, protons and electrons in amu (atomic mass units)<ul>
<li>Neutrons and protons have mass 1, electrons have a mass of something like 0.0005</li>
<li>Note that atomic mass is different from atomic weight - atomic weight is the weighted average of the atomic mass of the isotopes of an element</li>
<li>Relative atomic mass is a synonym for atomic weight, and this is rarely a whole number</li>
<li>Also note that neutrons and protons don't have exactly the same mass; we approximate</li>
<li>Furthermore, due to binding energy ...</li>
<li>But in any case the lesson here is that atomic mass and weight are rarely whole numbers for several reasons</li>
<li>The explanation in the lecture itself is a bit misleading, and doesn't really account for the difference between atomic mass and weight</li>
</ul>
</li>
<li>The periodic table, classifies elements in groups and families, primarily</li>
<li>Isotopes: different number of neutrons, but identical chemical properties (determined by protons and neutrons)<ul>
<li>However, due to the number of neutrons, can be unstable (i.e. radioactive - emit alpha, beta or gamma radiation from the nucleus)</li>
</ul>
</li>
<li>Chemical bonding: Aufbau principle, electrons fill up s orbitals before p orbitals<ul>
<li>Atoms can form bonds until their outermost shell is filled</li>
<li>Covalent bonds - sharing of electrons (to some degree), strong, predictable lengths and angles, can be double or TRIPLE<ul>
<li>Polar covalent bonds - when electrons are more attracted to one nucleus than the other, e.g. with water</li>
<li>This results in a dipole</li>
<li>On the other hand, in a C-C bond, for instance, the electrons are shared quite equally</li>
</ul>
</li>
<li>Ionic bonds - attraction of opposite charges (basically covalent but electronegativity difference is greater, usually &gt;1.8 - not in scope of course)</li>
<li>You'll learn, if you haven't already, that a lot of explanations in basic chem/bio courses are simplifications (i.e. lies)</li>
<li>van der Waal's forces: interactions of electrons of nonpolar substances</li>
<li>Hydrophobic interactions: nonpolar substances in the presence of polar substances</li>
<li>Bond energy: energy needed to separate two "bonded" atoms under physiological conditions (?)</li>
</ul>
</li>
<li>Electronegativity: tendency of an atom to attract electrons (when it occurs as part of a compound)<ul>
<li>Increases as atomic radius decreases and nuclear charge increases; ionization energy is similar, but slightly different</li>
</ul>
</li>
</ul>
<h2 class="header"><i>2</i>Large molecules<a class="headerlink" href="#large-molecules" name="large-molecules">&para;</a></h2>
<h3 class="header"><i>2.1</i>Acids and bases<a class="headerlink" href="#acids-and-bases" name="acids-and-bases">&para;</a></h3>
<ul>
<li>Acids donate hydronium ions (H+, simplified as hydronium ions)</li>
<li>Strong acids vs. weak acids - strong = release many H+ ions (simplified as, they release all their ions), weak = only some</li>
<li>Depends on pKa value; like covalent/ionic bonding, there is a continuum, but just pretend that what is said in this class is true</li>
<li>Bases accept H+, release OH-</li>
<li>Ex, NaOH is a strong base</li>
<li>Amino group (NH<sub>2</sub>): important part of many biological compounds (principal component of protein), weak base</li>
<li>Water: weak base AND acid, pH of pure water is 7, but almost all water is acidic due to dissolved ions, gases (e.g. CO<sub>2</sub>) etc</li>
<li>pH is concentration of hydrogen, measured in logarithms - pH of 7 ==&gt; 10<sup>-7</sup> mol / L</li>
<li>Buffers: make the solution resistant to pH change (reacts with added bases and acids)<ul>
<li>Can be composed of a weak acid and its conjugate base. The actual reactions are probably out of the scope of this course</li>
<li>Buffers can have different ranges over which they act</li>
<li>But once that range has been overshot, they no longer act as buffers</li>
</ul>
</li>
<li>The relation of functional groups to acid/base theory:<ul>
<li>Proteins are all amino acids - they have an amine group so they can accept, carboxyl group so they can donate</li>
<li>Others: ketones (O in the middle), aldehydes, etc</li>
<li>Side chains - ex, phosphates can add charges, sulfhydryl groups can combine, etc</li>
<li>Phosphate groups can store energy, release it; that's why they're part of the energy currency (ATP)</li>
</ul>
</li>
</ul>
<h3 class="header"><i>2.2</i>Isomerism<a class="headerlink" href="#isomerism" name="isomerism">&para;</a></h3>
<ul>
<li>Isomers: same chemical formula, different geometric structure<ul>
<li>Structural: different structural formula; a functional group is attached to a different carbon atom</li>
<li>Stereoisomers: same chemical formula, different arrangement of atoms in space<ul>
<li>Optical isomers (enantiomers), chiral compounds, mirror images of each other</li>
<li>Geometric isomers, ex cis &amp; trans isomers (different sides of a double bond, which cannot rotate)</li>
</ul>
</li>
</ul>
</li>
</ul>
<h3 class="header"><i>2.3</i>Components of a cell<a class="headerlink" href="#components-of-a-cell" name="components-of-a-cell">&para;</a></h3>
<ul>
<li>IN A CELL, WE HAVE:<ul>
<li>Lipids: good at forming barriers and compartments, as they resist the flow of aqueous solutions</li>
<li>Nucleic acids and proteins</li>
<li>Carbohydrates: energy storage, surface properties, rigid structure</li>
</ul>
</li>
<li>We are 70% water - most of the rest is biological polymers etc</li>
<li>WE ARE ALL MADE OF THE SAME FUNCTIONAL UNITS</li>
<li>Condensation reactions - one way to make a polymer<ul>
<li>Monomers, H + OH --&gt; H<sub>2</sub>O, then we get polymers</li>
</ul>
</li>
<li>Hydrolysis: breaking a polymer, consumes a molecule of water for every bond broken</li>
</ul>
<h3 class="header"><i>2.4</i>Sugars<a class="headerlink" href="#sugars" name="sugars">&para;</a></h3>
<ul>
<li>Carbohydrates: energy storage, building blocks for other molecules (nucleic acids)</li>
<li>Serve as structural components - insect exoskeletons are made of chitin, which is just modified sugar<ul>
<li>Attached to most membrane proteins, and through this give identity to cells by uniquely identifying their surface</li>
<li>For example, blood types determined by the type of sugar</li>
</ul>
</li>
<li>Ex: monosaccharides, disaccharides (2), oligosaccharides (3-20), polysaccharides (100+) (somehow 21-99 neglected)</li>
<li>General formula for a carbohydrate: CH<sub>2</sub>O<ul>
<li>Glucose: C<sub>6</sub>H<sub>12</sub>O<sub>6</sub></li>
<li>In polymers, of course, the formula is slightly different due to the loss of water</li>
</ul>
</li>
<li>When sugar dissolves - alpha and beta forms of ring structures, convert between them<ul>
<li>There is "handedness" in these systems, which is how you get flow in such systems</li>
</ul>
</li>
<li>Glyceraldehydes - 3 carbon sugars, important</li>
<li>Five-carbon sugars: ribose, deoxyribose, important in DNA</li>
<li>Six-carbon sugars: hexoses, the ones we consume and metabolise (alpha-manose, alpha-galactose, fructose which is not a true hexose)</li>
<li>Two glucose molecules can form 11 possible linkages<ul>
<li>However, enyzmes drive the formation of particular reactions, to ensure that only certain stable linkages are formed</li>
<li>Glycosidic linkage: formation of disaccharides through condensation reaction, ex sucrose from fructose and glucose</li>
<li>Maltose, cellobiose are structural isomers</li>
</ul>
</li>
<li>Plant cell walls: cellulose</li>
<li>Starch and glycogen (in food, e.g. potatoes; easily broken down)</li>
<li>Macromolecular structures:<ul>
<li>Linear (cellulose) - long, branched, stable - gives strength to plant walls</li>
<li>Curved, moderately branched (amylopectin) - also in plants</li>
<li>Curved and highly branched (glycogen, a polysaccharide) - gives strength, shape to cells</li>
</ul>
</li>
<li>Chitin: modified glucose, has an acetyl group attached to it (COCH<sub>3</sub>)</li>
</ul>
<h3 class="header"><i>2.5</i>Nucleic acids<a class="headerlink" href="#nucleic-acids" name="nucleic-acids">&para;</a></h3>
<ul>
<li>One of the four classes of macromolecules found exclusively in living organisms</li>
<li>Are polymers made of monomeric units (nucleotides)</li>
<li>Encode genes, specialised for the storage, transmission and use of genetic information</li>
<li>Can replicate to transmit the information to the next generation</li>
<li>Transcribed into RNA, which is translated into proteins</li>
<li>Central dogma of molecular biology:<ul>
<li>DNA can reproduce itself (replication)</li>
<li>And DNA can copy its information into RNA (transcription)</li>
<li>(which can then specify a sequence of amino acids in a polypeptide - translation(</li>
<li>Generally speaking, unidirectional - can't go from proteins to DNA usually (some viruses, RNA to DNA)</li>
<li>Life is just a long uninterrupted sequence of DNA replication</li>
</ul>
</li>
<li>Nucleotides also have roles in ATP and GTP - energy transducer and sources (as phosphate backbones are energy rich) in reactions<ul>
<li>Also in transmission of nervous signals</li>
</ul>
</li>
<li>Nucleotide: one or more phosphate groups, sugar, base<ul>
<li>Sugar: pentose, two flavours<ul>
<li>Either a hydroxyl group on one position (ribose, in RNA)</li>
<li>Or not such hydroxyl group (deoxyribose, in DNA)</li>
<li>This difference changes the backbone, etc</li>
</ul>
</li>
<li>Phosphate group: can be mono, di, or tri<ul>
<li>Bindings are very energy rich, when bindings are opened, lots of energy available</li>
</ul>
</li>
<li>Base nitrogen-containing single or double ring compounds<ul>
<li>Pyrimidines:  CT/U, smaller (single)</li>
<li>Purines: AG, larger (double)</li>
<li>Of course, hydrogen bonds between A-T (two bonds), C-G (three bonds, a little more stable)</li>
<li>Always one purine and one pyrimidine, so both bondings are the same size</li>
</ul>
</li>
</ul>
</li>
<li>Double-helical structure of DNA proposed by Watson and Crick; based on Rosalind Franklin's x-ray diffraction pattern (crystallography)<ul>
<li>Base inside, phosphate outside</li>
<li>Sugar molecule has 3 carbons; 5th has a phosphate group; always reacts with the 3rd carbon atom of the next one</li>
<li>So a 5' 3' connection, alternating</li>
<li>Polar molecule, etc</li>
<li>Cells can only synthesise DNA (polymerisation) in the 5' 3' direction, because of the direction of adding carbon?</li>
<li>Backbone: phosphate, sugar, alternating</li>
<li>The DNA backbone is negatively charged; bases are planar</li>
</ul>
</li>
<li>RNA: single stranded, uracil instead of thymine, single-stranded<ul>
<li>Bases still want to pair, however, so you get little loops etc, making for elaborate structures</li>
<li>RNA structures also have enzymatic capabilities</li>
</ul>
</li>
</ul>
<h3 class="header"><i>2.6</i>Lipids<a class="headerlink" href="#lipids" name="lipids">&para;</a></h3>
<ul>
<li>Nonpolar hydrocarbons, insoluble in water (due to nonpolar covalent bonds in carbon chain), like to clump together</li>
<li>Roles:<ul>
<li>Energy storage - most common (fats and oils)</li>
<li>Cell membranes (phospholipids)</li>
<li>Capture of light energy (carotenoids)</li>
<li>Also hormones and vitamins (steroids, modified fatty acids), thermal insulation, and electrical insulation of nerves</li>
</ul>
</li>
<li>Fats and oils: simple lipids (triglycerides), composed of glycerol (alcohol with 3 OH groups) plus 3 fatty acids, with ester linkage<ul>
<li>Condensation reaction: one fatty acid per OH group on the glycerol, they combine to form triglycerides</li>
<li>Saturated fatty acids: no double bonds, very straight and neatly packed, solid at room temp, mostly in animals (who need energy more)</li>
<li>Unsaturated: double bonds cause kinks (as carbon atoms aren't saturated by H), prevent tight packing, so liquid at room temp</li>
<li>Trans-fatty acids (geometric isomer, as opposed to cis)</li>
</ul>
</li>
<li>Phospholipids: similar structure to above, but phosphate group replaces one fatty acid<ul>
<li>Head has a positive charge, neck is negative, so hydrophilic head, hydrophobic tail</li>
<li>So both hydrophilic and hydrophobic ... forms a bilayer</li>
<li>The hydrophobic tails line up, and the heads face outward (on both sides), so a natural membrane - bilayer</li>
<li>Helps protect stuff from the outside world, naturally (no enzymes or anything needed)</li>
<li>Note that it's not a bond - just an aggregation, so things can travel through this membrane</li>
</ul>
</li>
<li>Carotenoids: light-absorbing pigments<ul>
<li>E.g. beta-carotene; symmetric molecule, when the body cuts it in half we get two vitamin A molecules</li>
<li>This is something that the body cannot synthesise itself, but important for vision, embryonic development etc</li>
</ul>
</li>
</ul>
<h3 class="header"><i>2.7</i>Proteins<a class="headerlink" href="#proteins" name="proteins">&para;</a></h3>
<ul>
<li>You can use the backbone of a protein to idealise the minimum volume a protein can occupy etc</li>
<li>The types of amino acids present are related to the 3-dimensional shapes of proteins</li>
<li>Views: backbone, sticks (electron orbitals), spheres (molecules), surface (nooks and crannies)</li>
<li>Every membrane in your body has some protein component</li>
<li>Professor Vogel's favourite amino acid (serine 360)<ul>
<li>Adding this amino acid to a protein changes the structure completely, changes the way cells divide</li>
<li>Because of a few additional charges added - electrostatic forces propagate through entire molecule</li>
</ul>
</li>
<li>Proteins: polymers of amino acids<ul>
<li>A few amino acids (called peptides, e.g. pheromones) to thousands of them (titin: 33,000 amino acids)</li>
<li>Folding is crucial to function of protein, largely (but not entirely) dictated by sequence of amino acids</li>
<li>Means we can make a prediction of the shape of protein based on its sequence of amino acids (which have different properties etc)</li>
<li>Can have many shapes, e.g. porin, lets things in and out, looks like a pretzel<ul>
<li>Whether or not something can go in doesn't depend only on shape - also on charge etc</li>
</ul>
</li>
</ul>
</li>
<li>20 amino acids: at the cytoplasmic pH (acidic), have lost hydrogen on carboxyl group, goes to other end<ul>
<li>amino group + carbon + side chain + carboxyl group, links to another one that does not have an extra H+<ul>
<li>The two form a peptide linkage - N terminal (NH3+ one) and C terminal (other one), for a protein - like poles</li>
<li>The side chain is the only specific part of an amino acid; the rest is the same</li>
<li>Determines how amino acids next to each other will behave (based on its charge)</li>
</ul>
</li>
<li>Results in a peptide backbone, repeating units: N-Cstuff-C-N-Cstuff-C.</li>
<li>The precise sequence of amino acids: primary structure</li>
<li>You can tell, from a protein's structure. what function it has; many categories</li>
</ul>
</li>
<li>Arginine, histidine, lysine - positively charged hydrophilic side chains</li>
<li>Aspartic acid, glutamic acid - negatively charged hydrophilic side chains</li>
<li>Serine, threonine, asparagine, gultamine, tyrosine: polar but uncharged hydrophilic side chains<ul>
<li>Tyrosine, threonine can be modified off hydroxyl group</li>
<li>Tyrosine has pi bonds, forms a 6-carbon aromatic side chain, affects neighbours by constraining them (stearic effects)</li>
</ul>
</li>
<li>Glycine: smallest amino acid, most flexibility, can adopt many different shapes (results in disordered parts of proteins)</li>
<li>Cysteine: sulfhydrl group can form disulfide bond, constrains the kind of shape protein can adopt</li>
<li>Proline: weird side chain, breaks things up</li>
<li>Alanine, Isoleucine, leucine, methionine, phenylalanine, tryptophan, valine: nonpolar hydrophobic side chains</li>
<li>Secondary structure: certain types of patterns will produce a different function, so proteins performing similar functions will have some similar regions<ul>
<li>Shape determined primarily by hydrogen bonds between the backbone of amino acids</li>
<li>Three most common ones: alpha helix, beta pleated sheet, and loops (can mess around here, to change shapes and enhance functionality)<ul>
<li>Alpha-helix: forms a straight rod, hydrogen bonds within rod formed by backbone, parallel to axis of helix</li>
<li>Side groups project outward from helix (even if internal)</li>
<li>Alpha helices can only go through plasma membranes if they have only hydrophobic amino acid side chains</li>
<li>Proteins can also form antiparallel coils - long, flexible, strong, can exert torquel so motor proteins usually have coils</li>
<li>Coiled coils arise when two alpha-helices have hydrophobic amino acids at every 4 positions (1 turn every 3.6 amino acids)</li>
<li>Beta pleated sheet: flat plates, ribbons ?</li>
<li>Strands can be parallel or antiparallel, can come from different polypeptides</li>
</ul>
</li>
</ul>
</li>
<li>Basic unit of proteins: polypeptides (amino acid chains) linked together<ul>
<li>Condensation reactions - carbon and nitrogens together, forms peptidyl bond (covalent)</li>
</ul>
</li>
<li>Polypeptide: chain of amino acid monomers, from N terminus until C terminus</li>
<li>Hydrogen bonding between amino acids: major driver in formation of higher-order structure from polypeptides</li>
<li>Easy to predict helices - characteristic set of amino acids and their position relative to each other<ul>
<li>Sheets, not as predictable (they're strong though)</li>
<li>Still, we can determine protein structure using crystallography, NMR etc</li>
</ul>
</li>
<li>Loops: usually between highly ordered structures (e.g. helices - common motif in organisms)<ul>
<li>Proline - fits neither in a helix nor a sheet because it makes a kink in the peptide</li>
<li>Due to its side chain, sort of makes a closed loop; N carries no H for hydrogen bonding</li>
<li>So I guess it results in a loop</li>
</ul>
</li>
<li>3D structure of proteins determined by:<ul>
<li>Location of covalent disulfide bridges<ul>
<li>Example: two cysteines can work together to make a covalent bond, forming a stable loop in the chain</li>
<li>Often loops move around a lot, but are fixed in relation to each other, due to covalent bonds</li>
</ul>
</li>
<li>Location of secondary structures<ul>
<li>E.g. parallel beta sheets buried, antiparallel usually on surface</li>
</ul>
</li>
<li>Ionic interactions between positive and negative charged side chains (form salt bridges)</li>
<li>Hydrophobic R groups aggregate, away from water (surface)</li>
<li>Forces between R groups (van der Waals for hydrophobic R, electrostatic for charged)</li>
</ul>
</li>
<li>Protein structure: made piece by piece<ul>
<li>First piece: always a methionine (basically the start amino acid, N-terminus)</li>
<li>Last piece (C-terminus): variable (can be anything), usually disordered</li>
<li>Shape of proteins - critical in how they communicate and link to each other</li>
</ul>
</li>
<li>Experimental evidence that primary structure specifies 3D structure:<ul>
<li>Ribonuclease, stable enzyme, added chemicals that disrupt hydrogen and ionic bonds (urea) and disulfide bridges</li>
<li>Also subject to temperature, low (?) pH, to completely denature the protein</li>
<li>Then, slowly remove the chemical agents ... the original 3D structure is restored, omg!!!</li>
<li>Note that this experiment depends on complete denaturation</li>
<li>Partial denaturation can be irreversible (e.g. boiling)<ul>
<li>For example, flipping hydrophobic regions to the surface and hydrophillic to the interior</li>
<li>Not all the bonds will be disrupted, so it can't regain its 3D structure after</li>
<li>Happens when you start cooking eggs - egg whites are partially denatured, yolk, not yet</li>
</ul>
</li>
</ul>
</li>
<li>Polypeptide: polymer of amino acids, linked by peptidyl bonds; in many cases, is the entire protein<ul>
<li>Many proteins composed of more than one polypeptide - e.g. hemoglobin</li>
<li>Protein domains: parts of a polypeptide with distinct functions</li>
<li>Kinase: enzyme, speeds up the process of transferring a phosphate to a hydroxyl group (so phosphylester?), negative charge</li>
<li>Activate themselves - phosphorylate (?), autoactivation = regulatory event, to change activity of a protein</li>
</ul>
</li>
<li>Electrostatic steering: how proteins join together, due to shape and surface chemistry (charges etc)</li>
<li>Chaperones: specialised proteins, keep temporarily exposed hydrophobic regions of other proteins from interacting inappropriately<ul>
<li>They sequester newly synthesised proteins, give them time to fold away from disruptive forces</li>
<li>Think of proteins as really attractive strippers trying to put their clothes on in a room full of people trying to take them off</li>
<li>Chaperones are essentially dressing rooms where strippers can put their clothes on</li>
<li>I don't know, similar to the in-class analogy</li>
<li>Structure of chaperone: protective cavity, nice physiological environment inside which proteins can fold</li>
<li>Example: HSP60 (heat shock protein 60, ubiquitous) - protect against thermal stress</li>
<li>Proteins can bind and enter (through electrostatic steering), fold, are released</li>
</ul>
</li>
</ul>
<h2 class="header"><i>3</i>Cellular membranes<a class="headerlink" href="#cellular-membranes" name="cellular-membranes">&para;</a></h2>
<h3 class="header"><i>3.1</i>Membrane structure<a class="headerlink" href="#membrane-structure" name="membrane-structure">&para;</a></h3>
<ul>
<li>Structure, due to phospholid bilayers: fluid mosaic model</li>
<li>Phospolipids form a sort of lake, fluid (no covalent bonds), proteins floating around</li>
<li>Four types of movement of phospholipids:<ul>
<li>Lateral diffusion: basically drifting sideways</li>
<li>Flexion: move left and right (individual I think)</li>
<li>Rotation</li>
<li>Flip-flop: upper and lower molecules switch places, rare because hydrophillic goes through hydrophobic interior</li>
</ul>
</li>
<li>Energetically favourable for bilayers to form an enclosed/sealed space<ul>
<li>So a sheet, yes, but with the edges all touching (natural configuration)</li>
<li>Offers an interesting glimpse into how cells evolved ... makes sense<br />
Membranes may vary in lipid composition</li>
</ul>
</li>
<li>Phospholipid chain length, degree of saturation, phosphate groups etc</li>
<li>Can even change seasonally, for example in hibernating animals, degree of saturation</li>
<li>Membranes can also contain cholesterol<ul>
<li>Cholesterol: steroid, also lipid (no hydrophillic head, only hydrophobic tail)</li>
<li>Very rigid, embedded in fatty acid to make the membrane more solid</li>
<li>Can counteract the effects of unsaturated fatty acids (the kinky ones)</li>
</ul>
</li>
<li>Can also contain proteins - the number and kind vary with cell function<ul>
<li>Peripheral membrane proteins: no hydrophobic regions, interact with hydrophilic heads or exposed parts of integral membrane proteins</li>
<li>Integral membrane proteins: partially or fully embedded, have both hydrophilic and hydrophobic domains</li>
<li>Again, no covalent bond here, just hydrophilic/hydrophobic interactions</li>
<li>Transmembrane proteins: anchored membrane proteins (polar proteins covalently attached to lipid), also has non-polar parts, can have loops</li>
</ul>
</li>
<li>Some membrane proteins can move freely within bilayer; others anchored to specific region<ul>
<li>Experiment: show that proteins embedded in membrane can diffuse freely within that membrane</li>
<li>One mouse cell, one human cell, both covered with membrane proteins</li>
<li>After a few hours, they sort of mix, resulting in one half-green, half red cell</li>
</ul>
</li>
<li>Membranes also have carbohydrates on outer surface - recognition sites for other cells and molecules<ul>
<li>Glycolipids: carbohydrate + lipid</li>
<li>Glycoproteins: carbohydrate + protein</li>
<li>Different combinations of these give cells an identity, like bar codes sort of; so other cells can recognise similar cells</li>
</ul>
</li>
</ul>
<h3 class="header"><i>3.2</i>Tissue formation<a class="headerlink" href="#tissue-formation" name="tissue-formation">&para;</a></h3>
<ul>
<li>Related to the above - similar cells recognise each other, come together</li>
<li>Example with sponges: pressed through filter, to separate into single cells</li>
<li>After a few hours, the same sponge is formed again - apparently the cells were able to come back together</li>
<li>Homotypic: most common, how tissues are formed; cells with the same molecule sticking out bind</li>
<li>Heterotypic: two cells with different proteins bind; example, sperm and ovum</li>
<li>Cell-cell junctions: specialised structures that hold cells together<ul>
<li>Tight junctions (occluding)</li>
<li>Cell-cell anchoring junctions - example, desmosomes</li>
<li>Channel forming junctions (gap junctions)</li>
<li>Example: epithelial cells, have all these junctions</li>
<li>Compartmentalised - have apical region for one task, basolateral region for another task</li>
<li>Tight junctions, in this case, seal the intercellular space, and separate apical from basolateral region</li>
<li>Desmosomes act like spot welds, like a plug, connecting cytoskeletons of neighbouring cells (e.g. cadherins - linked to cancer, as cells leave where they're supposed to be); keratin fibres</li>
<li>Gap junctions: allow communication, e.g. hydrophobic on outside, but polar on inside, so polar molecules can pass through, between cells (like an ethernet cable)</li>
</ul>
</li>
<li>Cell-matrix junctions: link cells to the extracellular matrix (keep them in place), done by integrins<ul>
<li>On the inside of cells, attached to actin filaments inside cell, extracellcular matrix outside</li>
<li>For cancer to metastasize, it would have to get rid of this connection</li>
</ul>
</li>
</ul>
<h3 class="header"><i>3.3</i>Selective permeability<a class="headerlink" href="#selective-permeability" name="selective-permeability">&para;</a></h3>
<ul>
<li>So membranes can control what passes through</li>
<li>Passive transport (diffusion - random movement toward equilibrium), no metabolic energy required<ul>
<li>Of course, at equilibrium, particles continue to move, but there is no net change in distribution (dynamic equilibrium)</li>
<li>Due to Brownian motion of individual molcules due to thermal motion, collisions</li>
<li>Diffusion rate depends on size of molecules, temperature of solution, and concentration gradient (difference between concentration in areas)</li>
</ul>
</li>
<li>Permeability - water and lipid-soluble molecules can diffuse easily<ul>
<li>Electrically charged and polar molecules, not so much</li>
<li>Hydrophobic move through the most easily</li>
<li>Small uncharged polar molecules, not as easily (water can, of course)</li>
<li>Large uncharged polar molecules, probably not</li>
<li>Ions - no way, need active transport</li>
<li>Osmosis: diffusion of water through membranes, passive, depends mostly on the number of solute particles present, not on type</li>
<li>Hypertonic: higher solute concentration in solution than cell, water will leave the cell and cell will shrink</li>
<li>Isotonic: same solute concentration as rest of cell (e.g. blood cells)</li>
<li>Hypotonic: lower solute concentration compared to cell; water will enter the cell, which will grow until it explodes</li>
<li>Animal cells placed in hypotonic solutions may burst; plant cells have rigid cell walls, build up turgor pressure, stop more water from entering</li>
<li>Why you would die when drinking 200g of NaCl but not 200g of glucose ... because glucose weighs more, so fewer particles, meh</li>
<li>Facilitated diffusion: for large polar and charged substances that aren't readily diffused<ul>
<li>Depends on channel proteins and carrier proteins</li>
<li>Channel proteins: hydrophilic pore, hydrophobic outside, polar things can go through; e.g. ligand gate ion channels (signal tells them to open) or voltage gate (when a certain voltage is reached)</li>
<li>Carrier proteins transport polar molecules across membranes in both directions, e.g. glucose, binds to carrier, comes through</li>
<li>Kind of like elevators I guess? Can be saturated, the number of carrier proteins limits the rate of diffusion</li>
</ul>
</li>
</ul>
</li>
<li>Active transport - pumped in or out, but energy needed; against concentration gradient<ul>
<li>Unidirectional: proteins involved, either uniporters, or sympoters, or antiporters</li>
<li>Unipoter: takes only one type of thing, sends it through</li>
<li>Sympoter: takes two things, moves them in same direction</li>
<li>Antiporter: two things, opposite directions (from opposite sites)</li>
<li>Primary active transport: (requires direct hydrolysis of ATP, on the spot, for that reaction)</li>
<li>Secondary active transport: energy comes from an ion concentration gradient, established by primary active transport</li>
<li>Example - sodium-potassium pump, oh god not this again<ul>
<li>Eventually you get lots of sodium outside, lots of potassium inside</li>
<li>When pumping out 3 sodiums and 2 potassium, uses the energy of ATP and builds up a positive charge outside</li>
<li>The pump controls the net movement of ions - one out</li>
<li>And so generates a membrane potential (electrical charge), omg nerve impulses</li>
<li>Also builds up a concentration gradient that allows other processes to occur</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 class="header"><i>4</i>Organisation of the cell<a class="headerlink" href="#organisation-of-the-cell" name="organisation-of-the-cell">&para;</a></h2>
<ul>
<li>Robert Hooke, first person to see a cell using a microscope (cork)</li>
<li>Cell theory:<ul>
<li>Cells are the fundamental units of life</li>
<li>All organisms are composed of cells (or maybe just one lol)</li>
<li>All cells come from pre-existing cells</li>
</ul>
</li>
<li>Implications of cell theory:<ul>
<li>Functions of cells are similar</li>
<li>Life is continuous</li>
<li>Origin of life = origin of cells</li>
</ul>
</li>
<li>Cells are usually small; exceptions:<ul>
<li>Bird eggs (each is one cell, and no, the nucleus is not the yolk; it's somewhere in the egg white)</li>
<li>Some neurons</li>
<li>Some algae</li>
<li>Bacteria cells</li>
</ul>
</li>
<li>But typically cells are small for a high surface area to volume ratio<ul>
<li>Volume determines the amount of chemical activity in the cell per unit time</li>
<li>Surface area determines the amount of substances that can pass the cell boundary</li>
<li>So reaction rate inside a cell is limited more by the surface area</li>
<li>As you need a lot of surface area in order to get all the shit you need to pass into the cell</li>
</ul>
</li>
</ul>
<h3 class="header"><i>4.1</i>Prokaryotic cells<a class="headerlink" href="#prokaryotic-cells" name="prokaryotic-cells">&para;</a></h3>
<ul>
<li>No nucleus obviously, although they have DNA; eubacteria and archaea</li>
<li>Cells very small; individuals are unicellular, but often found in chains or clusters (colonies etc)</li>
<li>Very successful - adapt to extreme conditions etc</li>
<li>Enclosed by plasma membrane</li>
<li>Thread of DNA, usually circular, contained in nucleoid</li>
<li>Cytoplasm: watery solution (cytosol), suspended particles</li>
<li>Ribosomes - site of protein synthesis (smaller than eukaryotic ribosomes)</li>
<li>Gram staining - thick peptidoglycan cell wall leads to gram positive, thin leads to negative<ul>
<li>Some antibiotics, such as penicillin, target the peptidoglycan cell wall specifically (so best against gram positive bacteria obviously)</li>
</ul>
</li>
<li>Some have capsule: gelatinous outer layer, prevents from drying out (and also protects from white blood cells sometimes)</li>
<li>Some special features developed by some bacteria:<ul>
<li>Photosynthetic bacteria have internal membrane system, contains the necessary molecules (e.g. cyanobacteria)</li>
<li>Others have internal membrane folds attached to plasma membrane - can function in cell division or energy-releasing reactions</li>
<li>Some have flagella, made of protein flagellin</li>
<li>Some have pili - hairlike structures projecting from surface, let bacteria stick to others (helpful for conjugation - sexual rep)</li>
<li>Some rod-shaped (bacilli) have cytoskeleton made of an actin-like protein</li>
</ul>
</li>
</ul>
<h3 class="header"><i>4.2</i>Eukaryotic cells<a class="headerlink" href="#eukaryotic-cells" name="eukaryotic-cells">&para;</a></h3>
<ul>
<li>Up to ten times larger than prokaryotes</li>
<li>Major advancement: compartmentalisation</li>
<li>Distinct nuclei enclosed in membrane, have membrane-bound organelles (like a little organ for the cell)</li>
<li>Endomembrane system: derived from outer membrane, has invaginated, complex structure<ul>
<li>This internal membrane system works efficiently, communicates etc</li>
<li>Lipid bilayers - vesicles can bud off, or fuse (like bubbles forming, unforming etc)</li>
<li>Also keeps cytosol protected from contents of vesicle or vice versa</li>
<li>Endoplasmic reticulum: network of interconnected membranes, connected to nuclear envelope; large surface area<ul>
<li>Rough ER: ribosomes attached, closer to nucleus; newly made proteins enter this lumen; modified, folded, transported to other regions via vesicles; most membrane-bound proteins synthesised here</li>
<li>Smooth ER: more tubular, no ribosomes (further from nucleus); stuff comes here to degrade; vesicles dock here and release drugs etc (oxidised here with a hydroxyl group, more soluble for excretion)</li>
<li>Smooth ERs are usually larger in drug users</li>
<li>They also synthesise lipids and steroids, degrade glycogen</li>
</ul>
</li>
<li>Golgi apparatus: stacks of flat membrane sacs (cisternae), midway between ER and membrane<ul>
<li>Distribution apparatus, like a post office (or airport, I LIKE THIS BETTER)</li>
<li>Vesicles coming in, fuse with cis region (close to ER)</li>
<li>Go inside lumen, so things are modified (e.g. glycoproteins), then released as vesicles on the trans side, transported to outside of cell</li>
<li>Two main roles: receives proteins from ER to further modify them (especially adding oligosaccharides to membrane lipids)</li>
<li>And concentrate, package, and sort proteins before sending them to their destination through the trans-region</li>
</ul>
</li>
<li>Lysosomes: vesicles containing digestive enzymes that come in part from the Golgi; usually quite acidic<ul>
<li>Think of them as the wandering stomachs of the cell; sites for breakdown of food and foreign material brought in by phagocytosis</li>
<li>They can also digest cell materials themselves (autophagy), ex: anorexics, cells don't have enough food, need to eat themselves</li>
<li>Also, when cell components are damaged and need to be removed and replaced by new ones</li>
<li>First, primary lysosome, fuses to form secondary lysosomes, which actually digest</li>
<li>Food is then secreted into the cytoplasm; unwanted stuff is ejected from cell</li>
</ul>
</li>
<li>Endomembrane system also includes plasma membrane and nuclear envelope</li>
<li>All these membranes communicate with each other via vesicles; are very collaborative</li>
</ul>
</li>
<li>Other things in animal cells:<ul>
<li>Centrioles (which make mitotic spindles for cell division)</li>
<li>Nucleus usually the largest organelle, holds DNA; replication and transcription happen here, as does assembly of ribosomes (in nucleolus)<ul>
<li>Nucleoplasm: chromatins ? Can't really hear what she's saying</li>
<li>Nuclear pores: control what goes in and out (DNA can't leave of course)</li>
<li>Nuclear localisation signal: binds to pore, signal for things being taken into the nucleus</li>
<li>In an interphase nucleus, chromatins are long, spread out, milky thing</li>
<li>When it is ready to divide, chromatin condenses, forms chromosomes</li>
</ul>
</li>
<li>Ribosomes: sites of protein synthesis (receive messenger RNA)<ul>
<li>Are in both prokaryotic and eukaryotic cells, similar structure in both</li>
<li>Consist of ribosomal RNA (rRNA), more than 50 different protein molecules</li>
<li>Can be found floating in cytoplasm, in the ER, in chloroplasts, in nucleus etc</li>
</ul>
</li>
<li>Mitochondria: power plants (make energy)<ul>
<li>Take the energy of fuel molecules (glucose), transform their energy into the bonds of energy-rich ATP (which cells can readily use)</li>
<li>The process of cellular respiration</li>
<li>Cells that require a lot of energy (e.g. muscle cells) have more mitochondria</li>
<li>Two membranes: outer membrane surrounds it</li>
<li>Inner membrane folds inward, to form cristae; large surface area for proteins involved in cellular respiration reactions</li>
<li>Space between membranes (matrix): contains ribosomes and mitochonrdrial DNA (different from nuclear DNA)</li>
</ul>
</li>
</ul>
</li>
<li>Some things specific to plant cells:<ul>
<li>Rigid cell wall, usually made of cellulose</li>
<li>Also plastids, only in plants and some pigments; e.g. chloroplasts, contain the pigment chlorophyll, double membrane, DNA, ribosomes<ul>
<li>Chloroplasts are stacked together like pitas - granum = stack of thylakoids</li>
<li>Other plastids: chromoplasts, contain red, orange, yellow pigments for flowers</li>
<li>Leucoplasts: storage, starch and fats (e.g. in potatoes)</li>
</ul>
</li>
<li>Vacuoles, in plant and protist cells: huge, store waste products and toxic compounds (dumpsters)<ul>
<li>Provide structure for plant cells - water enters by osmosis (due to high solute concentration), results in turgor pressure</li>
</ul>
</li>
</ul>
</li>
<li>Having all these organells has allowed diversification of functions, and eventual specialisation into tissues</li>
<li>How did eukaryotic cells evolve? Two main theories:<ul>
<li>Endomembrane system originated from invagination of plasma membrane (some bacteria have the beginnings of this)<ul>
<li>This would result in more compartmentalise, more surface area, etc</li>
</ul>
</li>
<li>Endosymbiosis theory: mitochondria and plastids arose when one cell engulfed another<ul>
<li>So mitochondria would have existed separately, until one day a bacterium swallowed one up</li>
<li>Same with cyanobacteria - engulfed, eventually reduced to a chloroplast</li>
<li>Evidence for this: mitochondria and cyanobacteria have their own DNA, which differ from eukaryotic DNA</li>
<li>Japan: discovery of a single-cell eukaryote, Hatena, ingested a green alga</li>
<li>The green alga lost most of its function, became reduced</li>
<li>Curiously, when the cell divides, one daughter cell is green, the other not</li>
<li>The other not-green one will go on to ingest another green alga, presumably out of jealousy</li>
</ul>
</li>
</ul>
</li>
</ul>
<h3 class="header"><i>4.3</i>The cytoskeleton<a class="headerlink" href="#the-cytoskeleton" name="the-cytoskeleton">&para;</a></h3>
<ul>
<li>Supports and maintains cell shape - but also dynamic (allows cells to change shape)</li>
<li>Holds organelles in position</li>
<li>Also imparts polarity to cells, so organelles can be moved from one point to another</li>
<li>Involved in cytoplasmic streaming</li>
<li>Interacts with extracellular structures to hold cell in place</li>
<li>Cells can flatten up on certain surfaces, become round on others; due to communication from outside<ul>
<li>When cells lose the ability to do this, they tend to become cancerous, oh no</li>
<li>This is because they lose contact adhesion</li>
</ul>
</li>
<li>Also present in prokaryotic cells</li>
<li>Have three components: (all formed of polymers; subunits are globular, dynamic proteins that harvest energy, usually ATP)<ul>
<li>Microfilaments: monomer (actin) forms trimer, which polymerises (triple string of pearls etc)<ul>
<li>Have a plus end and a minus end; grows more at plus end</li>
<li>So stuff can be moved around by placing it at the plus end, lol</li>
<li>Microfilaments usually at periphery of cell, near plasma membrane</li>
<li>Actin = ancient proteins, almost the same in plant and animal cells (?); actin and myosin in muscle cells result in contraction</li>
<li>Microfilaments also involved in formation of pseudopods (lol amoebae), when actin starts to grow out as a bundle, depolymerise and contract, pulling the cell forward as it does so</li>
<li>Cells move around by actin and myosin binding (reversible), energy cycle</li>
<li>This bind and release linked to action of a molecular lever at head (how cells move forward)</li>
<li>In some cells, microfilaments form meshwork just inside plasma membrane</li>
<li>Example: microvilli in human intestine, to increase surface area for absorption (not dynamic)</li>
</ul>
</li>
<li>Intermediate filaments: thicker than microfilaments, not as thick as microtubules (many polymers)<ul>
<li>Don't grow and shrink so much - stable network in cells</li>
<li>Animal cells have these; plants, yeast don't</li>
<li>Many different kinds - tough, ropelike protein assemblages, not dynamic, resist tension, anchor cell structures in place (esp. nucleus)</li>
</ul>
</li>
<li>Microtubules: separate chromosomes during cell division<ul>
<li>Long, hollow tubes, composed of the protein tubulin (dimer), plus and minus end</li>
<li>Also form a rigid internal skeleton in some cells, but dynamic (can form, unform, whatever)</li>
<li>Have + and - ends</li>
<li>Can change length rapidly by adding or loser dimers</li>
<li>Act as a framework for motor proteins, along with microfilaments</li>
</ul>
</li>
<li>They all work together to produce a dynamic system, allows cells to change shape, stay rigid, divide, etc</li>
</ul>
</li>
<li>Cilia and eukaryotic flagella, present on every cell<ul>
<li>Made up of microtubules in "9+2" array</li>
<li>They have the same arrangement; only difference is length (cilia shorter, usually many more)</li>
<li>Flagella are longer, usually one or two, movement is snakelike</li>
<li>Microtubules can be formed in basal bodies, which promote their organisation and assembly based on a template etc<ul>
<li>In the basal body, triplet organisation (9 triplets)</li>
</ul>
</li>
</ul>
</li>
<li>Centrioles: identical to basal bodies, except in a different place (near nucleus, not membrane) - formation of mitotic spindle</li>
<li>Motor proteins: undergo reversible shape changes, powered by ATP hydrolysis<ul>
<li>Dynein, binds to microtubule doublets, also moves chromosomes away from each other (for the mitotic spindle)</li>
<li>Nexin: can cross-link doublets, prevent them from sliding; cilium bends</li>
<li>Kinesin: binds to vesicle, helps move it along (cargo domain); usually kinesins work together, move up microtubule; long walks</li>
</ul>
</li>
<li>Experiments to determine the function of cellular components:<ul>
<li>Inhibition: drug that inhibits structure or process; does the function still occur?</li>
<li>Mutation: examine a cell that lacks the gene for the structure or process</li>
</ul>
</li>
</ul>
<h2 class="header"><i>5</i>Energy, enzymes, and metabolism<a class="headerlink" href="#energy-enzymes-and-metabolism" name="energy-enzymes-and-metabolism">&para;</a></h2>
<ul>
<li>Potential (chemical, concentration gradients, charge imbalances etc) vs. kinetic energy</li>
<li>Metabolism: sum total of all chemical reactions in an organism<ul>
<li>Anabolic: complex molecules made from simple; energy input required</li>
<li>Catabolic reactions: complex molecules broken down to simple; energy released</li>
</ul>
</li>
<li>It takes energy to impose order on a system; a lot of order in biological systems, comes at cost of lots of energy<ul>
<li>Total energy (enthalphy - H) = Gibbs free energy (G) + entropy (S) * temperature (T)</li>
</ul>
</li>
<li><span>$\Delta G$</span> between products and reactions - if negative, free energy released; if positive, energy consumed</li>
<li>If free energy is not available, no reaction occurs</li>
<li>Magnitude of G depends on delta H (total energy added or released) and change in entropy</li>
<li>Example: breaking apart ATP molecule to produce ADP; the high-energy bond is full of energy<ul>
<li>When you break protein into amino acids through hydrolysis, increase in entropy and thus -delta G scales with delta S</li>
</ul>
</li>
<li>First law: energy never destroyed etc</li>
<li>Second law: entropy increases through energy transformations so living organisms need constant supply of energy to maintain order</li>
<li>Exergonic reactions: release free energy (-deltaG): catabolism, complexity decreases (generates disorder)<ul>
<li>So products have less free energy than reactants (think of ball rolling down hill)</li>
</ul>
</li>
<li>Endergonic reactions: consume free energy (+deltaG), anabolism; complexity increases<ul>
<li>Think of pushing ball up a hill (products have more energy, so you need a certain amount of energy to initiate)</li>
</ul>
</li>
<li>In principle, chemical reaction can run in both directions<ul>
<li>So both the forward and reverse reaction are balanced, with deltaG = 0 at the equilibrium point</li>
<li>The value of deltaG gives you the point of equilibrium - near-zero values of deltaG relate to easily reversible reactions</li>
<li>If deltaG is very negative, will produce more product than reactant</li>
<li>If deltaG is very positive, will produce more reactant than product</li>
<li>Just think of it in terms of balls rolling down hills etc (ball will tend towards the bottom of the hill)</li>
</ul>
</li>
<li>Example: 100% glucose 1-phosphate initially, at equilibrium you have 95% glucose 6-phosphate (as 1 to 6 is exergonic)</li>
<li>So nothing ever really goes to completion (except asymptotically)</li>
<li>ATP: nucleotide adenosine triphosphate, captures and releases free energy (energy currency of cell)<ul>
<li>When hydrolysed (phosphorylester ? bond broken), releases much energy</li>
<li>Can phosphorylate, or donate phosphate groups, to other molecules</li>
<li>ATP, when it releases energy, is clearly very exergonic</li>
<li>Formation of ATP from ADP is endergonic; formation and breakdown reactions coupled</li>
<li>Remember the tertiary phosphate group, at the end</li>
</ul>
</li>
</ul>
<h3 class="header"><i>5.1</i>Enzymes<a class="headerlink" href="#enzymes" name="enzymes">&para;</a></h3>
<ul>
<li>Catalysts: speed up the rate of reaction; not themselves altered by reaction<ul>
<li>Do not make impossible reactions possible; just lower activation energy by providing an alternate reaction pathway</li>
</ul>
</li>
<li>Enzyme (protein): framework in which unfavoured reactions can take place, special kind of catalyst ?</li>
<li>Cells find ways to lower activation energy to make reactions proceed<ul>
<li>Transition states - unstable forms, higher free energy</li>
<li>Activation energy can come from heating a system (more kinetic energy)</li>
<li>Enzymes and ribozymes lower the energy barrier, bring reactants together, or stabilise things in states</li>
</ul>
</li>
<li>Biological catalysts (enzymes, ribozymes) highly specific - specific substrates (reactants) they use<ul>
<li>Substrate molecules bind to the active site of the enzyme</li>
<li>The 3D shape of the enzyme determines its specificity</li>
<li>Enyzme-substrate complex held together by electrical attraction etc, dynamic</li>
<li>Enzymes don't change final equilibrium or deltaG; just ensure that the reaction can start</li>
</ul>
</li>
<li>Mechanisms for catalysing reactions:<ul>
<li>Reducing dimensionality (orienting substrate molecules, bringing together atoms that will bond)</li>
<li>Physical strain (can stretch bonds in substrate molecules, making them unstable)</li>
<li>Chemical charge (can temporarily add chemical groups to substrates)</li>
</ul>
</li>
<li>Shape of enzyme active sites allows a specific substrate to fit (lock and key)<ul>
<li>Binding depends on hydrogen bonds, electrostatic attraction, hydrophobic interactions</li>
<li>Many enzymes change shape when they bind to the substrate - induced fit</li>
</ul>
</li>
<li>Some enzymes require partners:<ul>
<li>Prosthetic groups: non-amino acid groups, e.g. lipids bound to enzymes to stabilise</li>
<li>Cofactors: inorganic ions</li>
<li>Coenzymes: small carbon-containing molecules, not bound permanently to enzymes; e.g. ligands</li>
</ul>
</li>
<li>The rate of a catalysed reaction depends on substrate concentration<ul>
<li>Concentration of enzyme usually lower than concentration of substrate</li>
<li>At saturation, all enzymes are bound to substrate - maximum rate</li>
<li>Used to calculate enzyme efficiency - turnover (molecules of substrate converted per unit of enzyme)</li>
</ul>
</li>
<li>Reactions organised in metabolic pathways, interconnected both in space and in time</li>
<li>Inhibitors: regulate enzymes by binding to them and slowing reaction rates<ul>
<li>Naturally-occurring inhibitors regulate metabolism</li>
<li>For example, you're a bacterium, feasting on sugar, so you start metabolising</li>
<li>But then you run out of sugar ... so you need to stop metabolising otherwise you end up losing energy</li>
<li>Bacteria have a (negative?) feedback loop that can inhibit metabolism (raise activation energy somehow)</li>
<li>Irreversible inhibition: covalently bond to side chains in active site - permanently inactivates enzyme<ul>
<li>Example, nerve gas - once it goes in it never comes out</li>
</ul>
</li>
<li>Reversible inhibition: bonds noncovalently, prevents substrate from bonding temporarily<ul>
<li>For example, enzyme modified by another enzyme so that it can no longer catalyse reactions, geometrically</li>
</ul>
</li>
<li>Competitive inhibitors: compete with substrate, bind at the same active site</li>
<li>Noncompetitive inhibitors: same as above, but bind to the enzyme at a different site (change shape of enzyme, alters active site, prevent binding) - allostery<ul>
<li>Example: hemoglobin, quaternary structure, cooperative</li>
<li>Allosteric enzymes more sensitive to changes in concentration of substrate (graph: like acid-base titration or S versus log graph for nonallosteric)</li>
</ul>
</li>
</ul>
</li>
<li>Every enzyme is most active at a particular pH<ul>
<li>It influences the ionisation of functional groups</li>
<li>e.g. at low pH, COO- reacts with H+ to form COOH which is no longer charged; affects protein folding and thus enzyme function</li>
<li>Compartmentalisation is a way of safeguarding reactions etc, keeping them confined to a certain area (e.g. in the cell)</li>
</ul>
</li>
<li>Same thing with optimal temperature<ul>
<li>Active form: enzyme can bind to shit</li>
<li>Inactive form: enzyme cannot bind to shit</li>
</ul>
</li>
</ul>
<h2 class="header"><i>6</i>Pathways that release energy<a class="headerlink" href="#pathways-that-release-energy" name="pathways-that-release-energy">&para;</a></h2>
<ul>
<li>Being alive costs energy ... where does this energy come from etc (animals: food; plants: light)</li>
<li>Food/light is broken down (food) or built up (light) into fuel molecules<ul>
<li>This is GLUCOSE</li>
</ul>
</li>
</ul>
<h3 class="header"><i>6.1</i>Glucose oxidation<a class="headerlink" href="#glucose-oxidation" name="glucose-oxidation">&para;</a></h3>
<ul>
<li>Glucose: C<sub>6</sub>H<sub>12</sub>O<sub>6</sub> + 6O2 &amp;rarr; 6COs + 6H2O + free energy (deltaG is -686 kcal/mol)</li>
<li>Must be broken down in many progressive substeps (to prevent it from exploding I guess)</li>
<li>Each reaction is catalysed by a specific enzyme</li>
<li>In eukaryotes, metabolic pathways are compartmentalised in organelles (part of the reason for their efficiency)</li>
<li>Each pathway regulated by key enzymes</li>
<li>Metabolic pathways similar in all organisms<ul>
<li>Glycolysis: glucose is converted into pyruvate (only 3 carbon atoms) - 1:2</li>
<li>Cellular respiration (aerobic; needs oxygen present), converts pyruvate into H2O and CO2<ul>
<li>Complete oxidation, gives you 32 ATPs per glucose molecule; more efficient</li>
<li>Pathways: pyruvate oxidation and the citric acid cycle (Krebs cycle)</li>
<li>Pyruvate oxidation: broken down into 3 carbon dioxides; releases energy, either as ATP or electron carriers</li>
<li>Third pathway (respiration chain), the electron carriers from above are recycled (oxidation phosphorylation), generating ATPs</li>
<li>Energy transfer due to redox reactions (oil rig); oxidising agent makes the other one oxidised (so is reduced)</li>
<li>In glucose combustion, glucose is the reducing agent, O2 the oxidising agent</li>
</ul>
</li>
<li>OR fermentation (anerobic): converts pyruvate into CO2, and lactic acid or ethanol<ul>
<li>Incomplete oxidation, gives you two ATP per glucose</li>
<li>A lot of energy still trapped in the waste products (lactic acid and ethanol)</li>
</ul>
</li>
</ul>
</li>
<li>How is free energy harvested? Let's look at methane, the simplest carbohydrate<ul>
<li>Most reduced state corresponds with the highest free energy (most electrons)</li>
<li>Carbon more electronegative than hydrogen, so the electrons are closer to the carbon</li>
<li>But if you add oxygen (exchange a H), the carbon is oxidised and sort of loses an electron</li>
<li>The more oxidised, the less free energy</li>
<li>Scale: methane &amp;rarr; methanol &amp;rarr; methanal &amp;rarr; methanoic acid &amp;rarr; carbon dioxide</li>
<li>Carbon dioxide - most oxidised state, virtually no energy left (the C-H bond is very energy-rich however)</li>
<li>So in each oxidation step, free energy is released</li>
</ul>
</li>
<li>Ways of harvesting this free energy:<ul>
<li>Use to convert immediately from ADP to ATP (12 kcal/mol); less common</li>
<li>More efficiently: electron carriers are made to accept these electrons (50 kcal/mol of energy produced)<ul>
<li>Coenzyme NAD+ - key electron carrier in redox reactions</li>
<li>Add one H+ and 2 electrons, this reduces it into NADH</li>
<li>This reduced form (NADH) is then used in the respiratory chain (oxidised then, releasing energy) to produce ATP</li>
</ul>
</li>
</ul>
</li>
<li>Remember that at pretty much every step in cellular respiration, some ATP is produced; but only significantly so at the end<ul>
<li>Protongradient: drives oxidative phosphorylation, energy from oxidation of NAD and FAD (to produce ATP)</li>
</ul>
</li>
<li>These pathways all occur in different parts of the cell in eukaryotes<ul>
<li>Glycolysis and fermentation: outside mitochondria</li>
<li>Everything else, inside (respiratory chain in inner membrane, krebs and pyruvate oxidation in matrix)</li>
<li>In prokaryotes: cytoplasm has glycolysis, fermentation, krebs cycle; repiratory chain and pyruvate oxidation on plasma membrane</li>
</ul>
</li>
</ul>
<h3 class="header"><i>6.2</i>Aerobic pathways<a class="headerlink" href="#aerobic-pathways" name="aerobic-pathways">&para;</a></h3>
<ul>
<li>More on glycolysis: glucose broken down into 2 pyruvates<ul>
<li>First, you need to input energy to overcome activation energy of reaction - requires 2 ADP</li>
<li>Takes place in the cytosol</li>
<li>Generates no CO2, produces a small amount of energy</li>
<li>Net change: glucose becomes 2 molecules of pyruvate, and yu get 2 molecules of ATP and 2 NADH</li>
</ul>
</li>
<li>Pyruvate oxidation: links glycolysis and krebs cycle; occurs in mitochondrial matrix<ul>
<li>Pyruvate (3 carbons) oxidised to acetate (2 carbons) and carbon dioxide</li>
<li>NAD+ reduced to NADH, capturing some of the energy from the above oxidation</li>
<li>The remaining energy is stored by combining acetate and coenzyme A to form a huge complex (acetyl-coA)</li>
<li>Catalysed by pyruvate dehydrogenase (60 proteins, 5 co-enzymes) - this is incidentally what arsenic inhibits</li>
</ul>
</li>
<li>Acetyl CoA is the starting point of the citric acid cycle<ul>
<li>Inputs: water, acetyl CoA, water and electron carriers (NAD+, FAD, GDP)</li>
<li>Most of the electron carrier stuff etc done here (most of the energy transferred this way)</li>
<li>Outputs: CO2, reduced electron carriers, GTP (which converts ADP to ATP)</li>
<li>The acetyl CoA is hooked to a 4-carbon atom (oxaloacetate), forming citrate (citric acid), becomes isocitrate<ul>
<li>Loses first one carbon as carbon dioxide, then another</li>
</ul>
</li>
<li>Then regenerates to become oxaloacetate eventually (many intermediate steps, ATP etc produced)</li>
<li>Citric acid cycle is in steady state - concentration of intermediates don't change</li>
<li>Cycle continues when starting materials are available (reoxidised electron carriers and acetyl CoA)</li>
<li>Total yield: 6 carbon dioxide, 10 NADH, 2 FADH2, 4 ATP</li>
<li>Electron carriers need to be reoxidised so that they can take part in the cycle again, and to free the energy to make ATP<ul>
<li>Occurs either in oxidative phosphorylation (if O2 present) or fermentation (if anaerobic)</li>
</ul>
</li>
</ul>
</li>
</ul>
<h3 class="header"><i>6.3</i>Formation of ATP<a class="headerlink" href="#formation-of-atp" name="formation-of-atp">&para;</a></h3>
<ul>
<li>Oxidative phosphorylation: ATP synthesised using energy released by reoxidation of electron carriers</li>
<li>Extremely exergonic reaction - too much energy is produced, doesn't happen in cells</li>
<li>Instead, happens in little steps: electron transport and chemiosmosis</li>
<li>Electron transport:<ul>
<li>Happens in inner mitochondrial membrane</li>
<li>Electrons from NADH and FADH2 are passed through respiratory chain of membrane; energy set free at each step</li>
<li>This energy released is used to build up a proton gradient, protons pumped into inner membrane space<ul>
<li>Creates both a concentration gradient and a charge difference (potential energy - proton-motive force)</li>
<li>This force is converted into the production of ATP</li>
</ul>
</li>
<li>This series of membrane-associated protein complexes: large, protein complex (1-4), enzymes, etc</li>
</ul>
</li>
<li>Chemiosmosis<ul>
<li>Protons diffuse back into the mitochondrial matrix through the channel protein ATP synthase (can't pass through membrane)</li>
<li>When protons pass through it, the channel deforms, catalysing production of ATP</li>
<li>Results in ATP synthesis through diffusion ?</li>
<li>ATP leaves the mitochondria once made, which keeps the concentration low (drives the forward reaction - synthesis over hydrolysis)</li>
<li>Proton gradient is of course constantly maintained by electron transport and proton pumping</li>
<li>ATP synthesis can be uncoupled - if a different H+ diffusion channel is inserted into the mitochondrial membrane, energy set free lost as heat<ul>
<li>Useful for hibernating animals etc; H+ released as heat instead of coupled to ATP synthesis</li>
<li>Uncoupling protein: thermogenin</li>
</ul>
</li>
</ul>
</li>
<li>Experiment proving that an H+ gradient drives ATP synthesis<ul>
<li>Using pH of 9 in the inner membrane, pH of 7 on the outside</li>
<li>Resulted in the production of ATP</li>
</ul>
</li>
</ul>
<h3 class="header"><i>6.4</i>Fermentation<a class="headerlink" href="#fermentation" name="fermentation">&para;</a></h3>
<ul>
<li>Glycolysis is the same for aerobic and anaerobic pathways</li>
<li>But if you don't have oxygen, you might as well keep doing glycolysis to get a bit of ATP</li>
<li>However, you end up using all your NAD+ this way ... so fermentation is a way of regenerating NAD+ (which accepts electrons)</li>
<li>Fermentation occurs in the cytosol</li>
<li>NADH+ goes to NAD+ + H+ + 2e- (oxidation of NADH)<ul>
<li>Thus regenerating NAD+</li>
<li>More than one way of doing this - many fermentation pathways</li>
<li>Two major pathways: lactic acid fermentation and ethanol fermentation</li>
<li>Lactic acid fermentation:<ul>
<li>Pyruvate is reduced by the enzyme lactate dehydrogenase to produce lactate (going backwards in a way)</li>
<li>Which produces more NAD+</li>
<li>The end product is lactic acid (lactate), high-energy molecule but it stops there</li>
<li>This results in sore muscles (so when you use your muscles a lot, eventually supply of O2 dwindles, so muscles do this)</li>
</ul>
</li>
<li>Alcoholic fermentation:<ul>
<li>Occurs in yeast, some plant cells</li>
<li>Requires two reactions</li>
<li>First, decarboxylation reaction (results in aldehyde ethanal)</li>
<li>Ethanal is then reduced into ethanol</li>
<li>End result: just like 2 ATP, lol (incomplete combustion)</li>
<li>So obviously less efficient than cellular respiration, more of a short-term solution (long-term for yeast I guess)</li>
</ul>
</li>
</ul>
</li>
</ul>
<h3 class="header"><i>6.5</i>Metabolic pathways<a class="headerlink" href="#metabolic-pathways" name="metabolic-pathways">&para;</a></h3>
<ul>
<li>Glucose isn't the only thing involved in metabolic reactions</li>
<li>Other products enter metabolic pathways, all interrelated</li>
<li>Catabolic products enter the pathways at different steps</li>
<li>Intermediate products leave at some steps to be involved in other reactions</li>
<li>Lots of hustle and bustle</li>
<li>Catabolic interconversions:<ul>
<li>Polysacchardies are hydrolysed to glucose</li>
<li>Lipids broken down to glycerol and fatty acids (eventually acetyl CoA)</li>
<li>Proteins hydrolysed to amino acids, feed into glycolysis or krebs cycle</li>
<li>Most are reversible</li>
</ul>
</li>
<li>Anabolic interconversions:<ul>
<li>Gluconeogenesis: glucose formed from citric acid cycle and glycolysis intermediates, for storage (undo previous reactions?)</li>
<li>Acetyl CoA can be used to make fatty acids</li>
</ul>
</li>
<li>Glycolysis, krebs cycle, respiratory chain subject to allosteric regulation (effector molecule binds to site other than active site)<ul>
<li>Regulation by conformational change (not competitive binding)</li>
<li>Changed so that it can either bind or not bind, depending on step and purpose I guess</li>
<li>Negative feedback loop: if you have too much ATP, need to stop the cycle somehow; ATP can inhibit further production of itself</li>
<li>By blocking various enzymes at various points (potent inhibitor)</li>
<li>Low levels of ATP and high ATP: activator, to activate enzymes</li>
<li>Enough citrate can inhibit glycolysis, or activate fatty acid synthesis (too much acetyl CoA, get rid of it ... you get fat)</li>
<li>^ Example of a positive feedback loop</li>
</ul>
</li>
</ul>
<h2 class="header"><i>7</i>Photosynthesis<a class="headerlink" href="#photosynthesis" name="photosynthesis">&para;</a></h2>
<ul>
<li>Responsible for oxygen in the atmosphere, biomass, food etc</li>
<li>Carbon dioxide and water &amp;rarr; glucose, water and oxygen<ul>
<li>Light is required to split carbon dioxide molecules</li>
<li>Water is the source of the oxygen released (experiment done with radioactive tracers)</li>
</ul>
</li>
<li>Two pathways:<ul>
<li>Light reactions (light energy converted to chemical energy as ATP and NADPH)</li>
<li>Then carbon-fixation reactions (light independent), ATP from NADPH from above, plus carbon dioxide, produces carbohydrates</li>
</ul>
</li>
<li>Everything happens in the chloroplast</li>
<li>Photons are absorbed by molecules in this case</li>
<li>Molecules that absorb specific wavelengths in the visible range: pigments<ul>
<li>Example, chlorophyll absorbs red and blue light, and so leaves are green</li>
<li>Accessory pigments: cover the parts of spectrum that chlorophyll doesn't</li>
<li>Carotenoids and phycobilins; transfer the energy to chlorophylls</li>
<li>These pigments are found in chloroplasts, in thylakoids</li>
<li>When a pigment molecule absorbs a photon, the energy is usually transferred to another molecule<ul>
<li>Although it can be released as heat or light - fluorescence (lower wavelength)</li>
<li>Or it can use it to drive a chemical reaction, which also happens but not interesting in this context</li>
</ul>
</li>
<li>Target molecule must be near, with the right orientation and the right structure</li>
<li>Resonance energy transfer: electrons are excited from one molecule to the next (no electrons are transferred)</li>
<li>Chlorophyll becomes a reducing agent</li>
<li>The electron donated passed through chain of electron carriers onto NADP, which becomes NADPH and H+; ATP is made</li>
</ul>
</li>
<li>Electron transport:<ul>
<li>Noncyclic, produces NADPH and ATP, requires two photosystems<ul>
<li>Photosystem II results in water being split, due to a positive charge (?)</li>
<li>Light energy oxidises water, makes ATP</li>
<li>Photosystem I, different wavelength (more IR)</li>
<li>Positively charged pigment does not split water, accepts electron</li>
<li>The photosystems result in Z scheme</li>
</ul>
</li>
<li>Cyclic: ATP only</li>
<li>Calvin cycle: same as carbon fixation, water and carbon dioxide converted into organic compounds</li>
<li>Whatever</li>
</ul>
</li>
</ul>
	
    </div>
</div>

        </div>
    </div>
    <div id="footer" class="ui container">
        <div class="ui stackable grid">
            <div class="twelve wide column">
                <p>
                    Built by <a href="https://twitter.com/dellsystem">
                    @dellsystem</a>. Content is student-generated. <a
                    href="https://github.com/dellsystem/wikinotes">See the old codebase on GitHub</a>
                </p>
            </div>
            <div class="four wide right aligned column">
                <p><a href="#header">Back to top</a></p>
            </div>
        </div>
    </div>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-28456804-1', 'auto');
  ga('send', 'pageview');

</script>
</body>
</html>
